test/spec/services/homePagePreferenceSpec.js

```diff
@@ -8,20 +8,20 @@ describe("HomePagePreferenceService", function(){
     inject(function(_HomePagePreferenceService_){
       HomePagePreferenceService = _HomePagePreferenceService_;
     });
-    
+
```

Comment:
We should also double check that everyone is using the same `.editorconfig` so we have consistent white space settings.

Reply:
(just to avoid unnecessary diffs)

Reply:
@yapei FYI, it would be good to set up editorconfig for your editor

http://editorconfig.org/

Reply:
@yapei & thanks for taking the time to add these tests!  Quirk with `jasmine` aside, every little bit helps!

Reply:
@benjaminapetersen  @spadgett  I'm really sorry for the inconsistency in my PR, I should setup editorconfig before opening a PR as documented in our repo [before opening a PR](https://github.com/openshift/origin-web-console#before-opening-a-pull-request)

Reply:
No worries, not a big deal.  Appreciate it!